ui/components/app/import-token-link/import-token-link.component.js

```diff
@@ -21,14 +22,16 @@ export default function ImportTokenLink() {
 
   return (
     <Box className="import-token-link" textAlign={TEXT_ALIGN.CENTER}>
-      <Button
-        className="import-token-link__link"
-        type="link"
-        onClick={() => detectNewTokens()}
-      >
-        {t('refreshList')}
-      </Button>
-      {' or '}
+      {isMainnet && (
+        <Button
+          className="import-token-link__link"
+          type="link"
+          onClick={() => detectNewTokens()}
+        >
+          {t('refreshList')}
+        </Button>
+      )}
+      {isMainnet && ' or '}
```

Comment:
Looks like the `or` should be localized?

Reply:
Also repeating `isMainnet && ` can be avoided.